content/docs/specification/content.md

```diff
@@ -143,7 +143,7 @@ If a custom regex is configured with the `re=` field each match is used as a sub
 
 - Key: **from**
 - Permitted values: "simplified regex"
-- Example: "from=/$2/$1/$3"
+- Example: "from=/$2/$1/\$3"
```

Comment:
Why the change?

Reply:
Just like the last time in the previous PR. It's because of VSCode's Markdown extension.